### PR TITLE
add kujira dex and cw-staking adapters

### PR DESCRIPTION
**Pull request by eliseoci**

### CODE REVIEW

File: contracts/dex/src/exchanges/exchange_resolver.rs [hunk 1]
```diff
@@ -27,6 +29,8 @@ pub(crate) fn identify_exchange(value: &str) -> Result<&'static dyn Identify, De
         OSMOSIS => Ok(&Osmosis {
             local_proxy_addr: None,
         }),
+        #[cfg(feature = "kujira")]
```
Comment (Kayanski) [inline contracts/dex/src/exchanges/exchange_resolver.rs:32]: This doesn't need a feature flag, see how it's done here : https://github.com/AbstractSDK/adapters/blob/main/contracts/dex/src/exchanges/exchange_resolver.rs#L13

In order to allow for IBC interactions, the features should not prevent identification, but should prevent implementation if not on the right chain.

File: contracts/dex/src/exchanges/kujira.rs [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+impl Identify for Kujira {
+    fn name(&self) -> &'static str {
+        KUJIRA
+    }
+    fn over_ibc(&self) -> bool {
+        false
+    }
+}
+
+impl DexCommand for Kujira {
```
Comment (Kayanski) [inline contracts/dex/src/exchanges/kujira.rs:36]: Here you also need to only feature flag the DexCommand implementation part and not the Identify part (same reson, for IBC intergation"

File: contracts/dex/src/exchanges/exchange_resolver.rs (hunk 1 above)
Comment (eliseoci) [reply contracts/dex/src/exchanges/exchange_resolver.rs]: done, ty for your feedback

File: contracts/dex/src/exchanges/kujira.rs (hunk 2 above)
Comment (eliseoci) [reply contracts/dex/src/exchanges/kujira.rs:40]: done

**Review by Kayanski — APPROVED** [review]

**Pull request merged**